.env

```diff
@@ -0,0 +1,9 @@
+MYSQLDB_USER=root
+MYSQLDB_ROOT_PASSWORD=MichHMKSql
+MYSQLDB_DATABASE=book_app
+MYSQLDB_LOCAL_PORT=3306
+MYSQLDB_DOCKER_PORT=3308
+
+SPRING_LOCAL_PORT=8080
+SPRING_DOCKER_PORT=8088
+DEBUG_PORT=5005
```

Comment:
Remove .env from the remote repository. It's not secure. Add this file to .gitignore. Better push .env.template with variables without values